test/integration/lib/philipsHue_server.js

```diff
@@ -0,0 +1,83 @@
+var express = require ('express');
+var bodyParser = require('body-parser');
+var path = require('path');
+
+var philipsHue_server = (function() {
+
+	var welcome_page = '/html/philips_initial.html';		// Philips Hub webpage
+	var suc_resp = require('./success_response.js');		// TBD: move this to appropriate module
+	var api_resp = require('./json/bridge_status.json');	// full status response
+	var light_status_resp = require('./json/lights_status.json'); // light status response
```

Comment:
What is the advantage of requiring local modules here, compared to [there](https://github.com/fxbox/foxbox/pull/176/files#diff-aff726e949c461077c3a77d720cc4ad1R3) ?